docs/source/tutorial/highlevel.rst

```diff
@@ -57,3 +57,4 @@ Or to convert it to html and use layout analysis:
     >>> with open('samples/simple1.pdf', 'rb') as fin:
     ...     extract_text_to_fp(fin, output_string, laparams=LAParams(),
     ...                        output_type='html', codec=None)
+    >>> print(output_string.getvalue().strip())
```

Comment:
```suggestion
    >>> print(output_string.getvalue().strip())
        <html><head>
    <meta http-equiv="Content-Type" content="text/html">
    </head><body>
    <span style="position:absolute; border: gray 1px solid; left:0px; top:50px; width:612px; height:792px;"></span>
    <div style="position:absolute; top:50px;"><a name="1">Page 1</a></div>
    <div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:100px; top:122px; width:61px; height:24px;"><span style="font-family: Helvetica; font-size:24px">Hello 
    <br></span></div><div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:261px; top:122px; width:62px; height:24px;"><span style="font-family: Helvetica; font-size:24px">World
    <br></span></div><div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:100px; top:222px; width:61px; height:24px;"><span style="font-family: Helvetica; font-size:24px">Hello 
    <br></span></div><div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:261px; top:222px; width:62px; height:24px;"><span style="font-family: Helvetica; font-size:24px">World
    <br></span></div><div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:100px; top:322px; width:111px; height:24px;"><span style="font-family: Helvetica; font-size:24px">H e l l o  
    <br></span></div><div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:321px; top:322px; width:102px; height:24px;"><span style="font-family: Helvetica; font-size:24px">W o r l d
    <br></span></div><div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:100px; top:422px; width:111px; height:24px;"><span style="font-family: Helvetica; font-size:24px">H e l l o  
    <br></span></div><div style="position:absolute; border: textbox 1px solid; writing-mode:lr-tb; left:321px; top:422px; width:102px; height:24px;"><span style="font-family: Helvetica; font-size:24px">W o r l d
    <br></span></div><div style="position:absolute; top:0px;">Page: <a href="#1">1</a></div>
    </body></html>
```

We have test that compare the actual output to the expected output in the docs. I think adding this line will fix it.